ci-operator/config/openshift/telemeter/master.json

```diff
@@ -18,15 +18,18 @@
     "cluster": "https://api.ci.openshift.org",
     "namespace": "openshift",
     "name": "release",
-    "tag": "golang-1.9"
+    "tag": "golang-1.10"
```

Comment:
this doesn't have to match what's in the docker file?  I thought that dockerfile said 1.9

Reply:
it does not, i put you on 1.10 just to start.  This is effectively the "official" ci run